templates/addons/windows/kube-proxy-windows.yaml

```diff
@@ -57,7 +62,9 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
-        image: sigwindowstools/kube-proxy:${KUBERNETES_VERSION}-nanoserver
+        - name: USE_CI_VERSION
+          value: "${USE_CI_VERSION:false}"
```

Comment:
Maybe we can use `CI_VERSION` here instead so we don't have to define an extra environment variable in test/e2e/conformance_test.go

Reply:
updated